domain/appeal.go

```diff
@@ -128,8 +128,12 @@ func (a Appeal) ToGrant() (*Grant, error) {
 		if err != nil {
 			return nil, fmt.Errorf("parsing duration %q: %w", a.Options.Duration, err)
 		}
-		expDate := time.Now().Add(duration)
-		grant.ExpirationDate = &expDate
+		if duration == 0*time.Second {
```

Comment:
do we need the `time.Second` ? or `if duration == 0 {` is enough?

Reply:
@mabdh thanks, done